.gitignore

```
@@ -1,3 +1,5 @@
 book
-.vale.ini
-.vale
+.vale/styles/Espressif-latest
+.vale/styles/Google
+.vale/styles/Microsoft
+.vale/styles/RedHat
```

I believe you should be able to do something like this:

```suggestion
.vale/styles/*
!.vale/styles/Rust/
```